src/group_theory/subgroup.lean

```diff
@@ -1281,6 +1281,15 @@ lemma cod_restrict_apply {G : Type*} [group G] {N : Type*} [group N] (f : G →*
   (S : subgroup N) (h : ∀ (x : G), f x ∈ S) {x : G} :
     f.cod_restrict S h x = ⟨f x, h x⟩ := rfl
 
+@[to_additive] lemma range_subgroup_of_eq_of_le {G₁ G₂ : Type*} [group G₁] [group G₂]
```

Comment:
This is actually a lemma about the `subgroup_of` construction, applied to the `range` of a function, so the "of" should stay.  But come to think of it, maybe the name should be `subgroup_of_range_eq_of_le` rather than `range_subgroup_of_eq_of_le`?

Reply:
Ah, yes, I was confused. Indeed, `subgroup_of_range_eq_of_le` is probably better. Do as you prefer, I'll trust you on this.